src/Resources/RefreshViews.sql

```diff
@@ -0,0 +1,8 @@
+﻿DECLARE @sql NVARCHAR(MAX)
```

Comment:
Podľa mňa by tento súbor mal byť pridaný do `.csproj` ako embdebed resource. Podobne ako `MigrationsHistoryTableScript`. Mám pocit, že si to aj robile, lebo ti to nefungovalo. Ale teraz to tu nevidím.